app/src/main/java/org/sopt/dosopttemplate/features/util/BindingAdapter.kt

```diff
@@ -0,0 +1,21 @@
+package org.sopt.dosopttemplate.features.util
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import coil.load
+import coil.transform.RoundedCornersTransformation
+import org.sopt.dosopttemplate.R
+
+@BindingAdapter("imageUrl")
+fun loadImage(view: ImageView, url: String?) {
+    view.load(url)
+}
+
+@BindingAdapter("setCircleImage")
+fun ImageView.setCircleImage(img: String?) {
+    val imageResource = if (img.isNullOrEmpty()) R.drawable.img_empty_profile_image else img
+
+    load(imageResource) {
+        transformations(RoundedCornersTransformation(60f))
+    }
```

Comment:
이게 이미지 둥글게 하는 코드인건가??
신기하군

Reply:
마자요 coil 체고

Reply:
```suggestion
@BindingAdapter("setCircleImage")
fun ImageView.setCircleImage(img: String?) {
    val imageResource = if (img.isNullOrEmpty()) R.drawable.img_empty_profile_image else img

    load(imageResource) {
        transformations(RoundedCornersTransformation(둥근 정도))
    }
```
coil 라이브러리를 이용해서 둥글게 하는 코드 맞습니다! 둥근 정도도 조절할 수 있어요!

Reply:
엇 이거 하려다 못했는데..!! 꿀팁공유보고 참고해볼게여

Reply:
박소현 그만 잘해라 ㅋ.ㅋ